src/networking/utility/ClientInfo.java

```diff
@@ -0,0 +1,24 @@
+package networking.utility;
+
+/**
+* 
+* @author Prudhvi Vardhan Reddy Pulagam
+* 
+* @ClientInfo class consists of information about respective individual
+* client's IP addres and Port number on which the application listening too. 
+*
+* @getIp() method returns client's IP address as String and takes no arguments.
+* @getPort() method returns client's Port number as integer and takes no arguments
```

Comment:
This way of commenting is not being followed by us. Please cross-check and do the corrections.